javalib/src/main/scala/java/math/BigInteger.scala

```diff
@@ -51,6 +53,8 @@ object BigInteger {
 
   final val ONE = new BigInteger(1, 1)
 
+  final val TWO = new BigInteger(1, 2)
```

Comment:
I think after all my comments, this is not needed anymore.

(It's a real JDK API which could be submitted upstream as a separate PR, but there's no reason to add it here.)